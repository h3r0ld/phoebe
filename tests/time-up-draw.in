start map.txt
hardworking-little-robot-random off
set-clock 99
end-turn
end-turn
quit