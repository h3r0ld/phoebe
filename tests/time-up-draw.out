Player 0: Cell(22,8) Speed: 2 CanChangeDirection: true
Player 1: Cell(22,10) Speed: 2 CanChangeDirection: true
Player 0: Distance - 0; Player 1: Distance - 0; Winner: Player 1
Player 0: Cell(22,8) Speed: 2 CanChangeDirection: true